tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     vmImage: 'macOS-10.13'
   steps:
   
-    - template: templates/set-test-data-variables-step.yml
+    - template: templates/mac-set-variables-and-download.yml
     - template: templates/set-version-number-variables-step.yml
```

Comment:
Should remove --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) from line 62 below, to avoid 2 model downloads.